schema.graphql

```diff
@@ -553,3 +612,22 @@ type ReferralCode @entity {
     "total number of registered referees"
     numReferees: BigInt!
 }
+
+# No such contract event, this is artificial for monitoring only
+type BadDebtHappened @entity {
```

Comment:
Here is the only place I see `BadDebtHappened` in the repo. Should we remove this?

Reply:
Yea we didn't use it. Just keep it for backward compatibility.